scripts/travis/travis_build_and_test.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Script called by Travis to do a CPU-only build of and test Caffe.
+# Script called by Travis to build and test Caffe.
```

Comment:
To be clear the Travis CI tests are CPU-only and this should stay in the comments. The GPU code is compiled but not tested for lack of hardware.